ods-setup/repos.sh

```diff
@@ -27,6 +27,7 @@ BITBUCKET_ODS_PROJECT="opendevstack"
 SYNC="n"
 
 function usage {
+  printf "NOTE: Before you run this script, make sure to have at least Git 2.7 in your system.\n\n"
```

Comment:
I think it would be nicer to have the same Git version everywhere to avoid confusion. So even though this may run with 2.7, let's require 2.13 everywhere so people update their Git installations. 2.13 was released May 2017, so is already over 3 years old.

Reply:
True, better to be consistent and going for the higher version will be safer